### PR TITLE
It is ejabberd

### PR DESCRIPTION
**Pull request by Neustradamus**
Changes about ejabberd.

### CODE REVIEW

File: root/etc/e-smith/templates/etc/dnsmasq.conf/80xmpp_srv_record [hunk 1]
```diff
@@ -2,5 +2,5 @@
 # 80xmpp_srv_records
 #
 srv-host=_xmpp-client._tcp.{$DomainName},{$SystemName . '.' . $DomainName},5222,0,5
-srv-host=_xmpp-client._tcp.{$DomainName},{$SystemName . '.' . $DomainName},5223,0,5
+srv-host=_xmpps-client._tcp.{$DomainName},{$SystemName . '.' . $DomainName},5223,0,5
```
Comment (DavidePrincipi) [inline root/etc/e-smith/templates/etc/dnsmasq.conf/80xmpp_srv_record]: Do you have any reference to `xmpps-client` service name definition? It seems only `xmpp-client` is [standardized](https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml?search=5222)

I propose to remove that line.

```suggestion
```

**Comment by DavidePrincipi** [comment]
Thank you for your patches.

I reverted the changes affecting code files, retaining your modifications of UI labels and README. I hope this is good!

File: root/etc/e-smith/templates/etc/dnsmasq.conf/80xmpp_srv_record (hunk 1 above)
Comment (Neustradamus) [reply root/etc/e-smith/templates/etc/dnsmasq.conf/80xmpp_srv_record]: -> xmpps-client for 5223: https://xmpp.org/extensions/xep-0368.html

**Comment by Neustradamus** [comment]
Why some reverts?

-> My commits were with Ejabberd in filename instead of Ejabber...

Ejabber_Title -> Ejabberd_Title
Ejabber_Description -> Ejabberd_Description
Ejabber_Tags -> Ejabberd_Tags
class Ejabber -> class Ejabberd

A missing "_":
EjabberdURL -> Ejabberd_URL

A space in more:
//Retrieve the  ejabberd URL -> //Retrieve the ejabberd URL

Ejabberd -> ejabberd:
echo '[WARNING] Ejabberd is not running, backup is not possible' -> echo '[WARNING] ejabberd is not running, backup is not possible'
<a href='$url' target='_blank'>Ejabberd</a><br/> -> <a href='$url' target='_blank'>ejabberd</a><br/>

Please note that all have been done in other part:
- https://github.com/NethServer/nethserver-fail2ban/pull/74
- https://github.com/NethServer/nethserver-lang/pull/27

**Comment by gsanchietti** [comment]
> In more:
> Can you change in the same time: jabberadmins to xmppadmins

No, because it is a breaking change: updated machines which already use `jabberadmins` will not work anymore.
We can plan such changes only on major release (eg. NethServer 7.8), but there should be an automated (or at least manual) migration path.

**Comment by Neustradamus** [comment]
@gsanchietti: It is for this that I have not done commit about it ;)

But can you see my other points?
Values, filenames, ...

**Comment by gsanchietti** [comment]
> But can you see my other points?

Not really :) I agree with the UI label changes which should match the upstream project name.

But I see no improvements on changing filenames and code variables on a consolidated code base, only possible regressions. For we will take into account such request for future works. ;)

**Comment by DavidePrincipi** [comment]
I think issuing a code refactor now just to fix the `ejabberd` name is risky and might introduce some regressions. 

As said, I prefer to limit the changes to UI labels.

Thanks again for your fixes that already lead to many improvements. Let's discuss further improvements on community ;)

**Comment by Neustradamus** [comment]
What regressions, there are no problems:
- Filenames are correctly changed in code and translations too.

Please look again all in my previous message, for examples:
- A missing "_" in one.
- A space " " in more.

It is a warning message:
- [WARNING] Ejabberd is not running, backup is not possible -> [WARNING] ejabberd is not running, backup is not possible
- [WARNING] Ejabberd is not running, restore is not possible -> [WARNING] ejabberd is not running, restore is not possible

**Comment by Neustradamus** [comment]
I will do new PRs :)

**Pull request merged**